CI failure is unrelated:

```
[2021-04-15T05:44:49.627Z] #22 53.63       aarch64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.7m -c /tmp/pip-install-ymd4jk19/Cython/Cython/Plex/Scanners.c -o build/temp.linux-aarch64-3.7/tmp/pip-install-ymd4jk19/Cython/Cython/Plex/Scanners.o

[2021-04-15T05:44:49.627Z] #22 53.63       /tmp/pip-install-ymd4jk19/Cython/Cython/Plex/Scanners.c:21:10: fatal error: Python.h: No such file or directory

[2021-04-15T05:44:49.627Z] #22 53.63        #include "Python.h"

[2021-04-15T05:44:49.627Z] #22 53.63                 ^~~~~~~~~~

[2021-04-15T05:44:49.627Z] #22 53.63       compilation terminated.

[2021-04-15T05:44:49.627Z] #22 53.63       error: command 'aarch64-linux-gnu-gcc' failed with exit status 1

[2021-04-15T05:44:49.627Z] #22 53.63   

[2021-04-15T05:44:49.627Z] #22 53.63       ----------------------------------------

[2021-04-15T05:44:49.627Z] #22 53.63   Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ymd4jk19/Cython/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-mtj_1y4q/install-record.txt --single-version-externally-managed --prefix /tmp/pip-build-env-t7_k5vse --compile" failed with error code 1 in /tmp/pip-install-ymd4jk19/Cython/

[2021-04-15T05:44:49.627Z] #22 53.63   

[2021-04-15T05:44:49.627Z] #22 53.63   ----------------------------------------

[2021-04-15T05:44:49.627Z] #22 53.74 Command "/usr/bin/python3 -m pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-t7_k5vse --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel Cython" failed with error code 1 in None

[2021-04-15T05:44:49.891Z] #22 ERROR: executor failed running [/bin/sh -c pip3 install yamllint==1.16.0]: exit code: 1
```

--- a/hack/dockerfile/install/rootlesskit.installer
+++ b/hack/dockerfile/install/rootlesskit.installer
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-# v0.14.1
-: "${ROOTLESSKIT_COMMIT:=ed9b8c5cc48d29d0a979dae52a24f6e886795abd}"
+# v0.14.2
+: "${ROOTLESSKIT_COMMIT:=4cd567642273d369adaadcbadca00880552c1778}"
 
 install_rootlesskit() {
 	case "$1" in

